internal/config/config.go

```diff
@@ -290,7 +293,13 @@ var agentConfig = `
   ## Log only error level messages.
   # quiet = false
 
-  ## Log file name, the empty string means to log to stderr.
+  ## Log target controls the destination for logs and can be one of "file",
+  ## "stderr" or, on Windows, "eventlog".  When set to "file", the output file
+  ## is determined by the "logfile" setting.
+  # logtarget = "file"
+
+  ## Log file name when "logtarget" is set to file.  If set to the empty string
+  ## logs will be sent to to stderr.
```

Comment:
I am assuming this also means if logtarget == "file" AND logfile == "", then stderr (default) is the one used ?

Reply:
Yes.  I'll fix the grammar error here too.

Reply:
In 1.13 the recommended setup for stderr will be only:
```
logtarget = "stderr"
```

The recommended setup for file logging will be:
```
logtarget = "file"
logfile = "/var/log/telegraf/telegraf.log"
```

The idea is that `logtarget` should be used to control where logs go, and `logfile` will only control the name of the filename.